lib/lita/handlers/irkit.rb

```diff
@@ -48,30 +47,30 @@ def ir_send_all_off(response)
 
       def ir_unregister(response)
         cmd = response.matches[0][0]
-        Lita.redis.del key_for_cmd(cmd)
+        redis.del cmd
         response.reply ":ok_woman:"
       end
 
-    private
+      def ir_migrate(response)
+        keys = Lita.redis.keys('irkit:messages:*')
 
-      def key_for_cmd(cmd)
-        "irkit:messages:#{cmd}"
-      end
+        Lita.redis.pipelined do
+          keys.each do |key|
+            Lita.redis.rename key, key.sub(/^irkit:messages:/, 'handlers:irkit:')
```

Comment:
I'd like to use `handlers:irkit:messages:` to enable to store `handlers:irkit:schedules:` or something in the future. May be I change it after merging this pull-request.